[2024-08-09T09:21:54.017+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-09T09:21:54.210+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Weather_Data_Extraction.generate_output_file manual__2024-08-09T09:21:51.094913+00:00 [queued]>
[2024-08-09T09:21:54.219+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Weather_Data_Extraction.generate_output_file manual__2024-08-09T09:21:51.094913+00:00 [queued]>
[2024-08-09T09:21:54.219+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-09T09:21:54.232+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): generate_output_file> on 2024-08-09 09:21:51.094913+00:00
[2024-08-09T09:21:54.244+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1755) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-09T09:21:54.246+0000] {standard_task_runner.py:63} INFO - Started process 1757 to run task
[2024-08-09T09:21:54.247+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Weather_Data_Extraction', 'generate_output_file', 'manual__2024-08-09T09:21:51.094913+00:00', '--job-id', '116', '--raw', '--subdir', 'DAGS_FOLDER/Data_Weather_Extraction.py', '--cfg-path', '/tmp/tmpt08barfq']
[2024-08-09T09:21:54.250+0000] {standard_task_runner.py:91} INFO - Job 116: Subtask generate_output_file
[2024-08-09T09:21:54.263+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-08-09T09:21:54.297+0000] {task_command.py:426} INFO - Running <TaskInstance: Weather_Data_Extraction.generate_output_file manual__2024-08-09T09:21:51.094913+00:00 [running]> on host ubuntu
[2024-08-09T09:21:54.404+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ducky' AIRFLOW_CTX_DAG_ID='Weather_Data_Extraction' AIRFLOW_CTX_TASK_ID='generate_output_file' AIRFLOW_CTX_EXECUTION_DATE='2024-08-09T09:21:51.094913+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-09T09:21:51.094913+00:00'
[2024-08-09T09:21:54.406+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-09T09:21:54.421+0000] {logging_mixin.py:188} INFO - Generated output file: Data_Weather-20240808.csv
[2024-08-09T09:21:54.421+0000] {logging_mixin.py:188} INFO - Execution Date: 2024/08/08
[2024-08-09T09:21:54.447+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-09T09:21:54.448+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-09T09:21:54.457+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Weather_Data_Extraction, task_id=generate_output_file, run_id=manual__2024-08-09T09:21:51.094913+00:00, execution_date=20240809T092151, start_date=20240809T092154, end_date=20240809T092154
[2024-08-09T09:21:54.500+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-09T09:21:54.536+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-09T09:21:54.538+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
